libbeat/docs/shared-ssl-config.asciidoc

```diff
@@ -145,21 +145,28 @@ suites are used (recommended).
 
 The following cipher suites are available:
 
-* RSA-RC4-128-SHA (disabled by default - RC4 not recommended)
-* RSA-3DES-CBC3-SHA
-* RSA-AES-128-CBC-SHA
-* RSA-AES-256-CBC-SHA
-* ECDHE-ECDSA-RC4-128-SHA (disabled by default - RC4 not recommended)
 * ECDHE-ECDSA-AES-128-CBC-SHA
+* ECDHE-ECDSA-AES-128-CBC-SHA256 (TLS 1.2 only, disabled by default)
+* ECDHE-ECDSA-AES-128-GCM-SHA256 (TLS 1.2 only)
 * ECDHE-ECDSA-AES-256-CBC-SHA
-* ECDHE-RSA-RC4-128-SHA (disabled by default- RC4 not recommended)
+* ECDHE-ECDSA-AES-256-GCM-SHA384 (TLS 1.2 only)
+* ECDHE-ECDSA-CHACHA20-POLY1305 (TLS 1.2 only)
+* ECDHE-ECDSA-RC4-128-SHA (disabled by default - RC4 not recommended)
 * ECDHE-RSA-3DES-CBC3-SHA
 * ECDHE-RSA-AES-128-CBC-SHA
-* ECDHE-RSA-AES-256-CBC-SHA
+* ECDHE-RSA-AES-128-CBC-SHA256 (TLS 1.2 only, disabled by default)
 * ECDHE-RSA-AES-128-GCM-SHA256 (TLS 1.2 only)
-* ECDHE-ECDSA-AES-128-GCM-SHA256 (TLS 1.2 only)
+* ECDHE-RSA-AES-256-CBC-SHA
 * ECDHE-RSA-AES-256-GCM-SHA384 (TLS 1.2 only)
-* ECDHE-ECDSA-AES-256-GCM-SHA384 (TLS 1.2 only)
+* ECDHE-RSA-CHACHA20-POLY1205 (TLS 1.2 only)
+* ECDHE-RSA-RC4-128-SHA (disabled by default- RC4 not recommended)
+* RSA-3DES-CBC3-SHA
+* RSA-AES-128-CBC-SHA
+* RSA-AES-128-CBC-SHA256 (TLS 1.2 only, disabled by default)
+* RSA-AES-128-GCM-SHA256 (TLS 1.2 only)
+* RSA-AES-256-CBC-SHA
+* RSA-AES-256-GCM-SHA384 (TLS 1.2 only)
+* RSA-RC4-128-SHA (disabled by default - RC4 not recommended)
```

Comment:
I sorted the list, so to not loose all sanity when double checking suite names + properties.

Reply:
Good thing!